src/main/java/nl/ordina/jobcrawler/controller/SkillMatcherController.java

```diff
@@ -14,22 +13,18 @@
 @RestController
 public class SkillMatcherController {
 
-
     private final SkillMatcherService skillMatcherService;
 
     @Autowired
-    public SkillMatcherController(
-            SkillMatcherService skillMatcherService) {
+    public SkillMatcherController(SkillMatcherService skillMatcherService) {
         this.skillMatcherService = skillMatcherService;
     }
 
-
     // rematch the skills
-    @PutMapping(path = "skillmatcher")
-    public ResponseEntity<ResponseCode> relinkSkills() {
+    @PutMapping(path = "/skillmatcher")
+    public void relinkSkills() {
         log.info("relink skills");
         Thread newThread = new Thread(skillMatcherService::relinkSkills);
         newThread.start();
-        return new ResponseEntity<>(new ResponseCode("OK"), HttpStatus.OK);
```

Comment:
This method was changed from returning this body 
```
{
  "errorCode": "OK"
}
```
on which the frontend verifies if the function ran correctly, to returning no body as this is a more standard way. 
I think it is better to adjust the frontend to this.

Reply:
The frontend actually doesn't do anything with the response (I do get an error in console, but the put request is still being processed by the backend). But judging by the logs in IntelliJ and looking into the vacancy_skills table I don't see any link being made.

Reply:
Just tested it and the problem is the findByOrderByNameAsc which was not included in the this controller. I am working on adding it.